sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -19,23 +20,27 @@
 from feast.field import Field
 from feast.infra.online_stores.online_store import OnlineStore
 from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
+from feast.protos.feast.types.Value_pb2 import FloatList
 from feast.protos.feast.types.Value_pb2 import Value as ValueProto
 from feast.repo_config import FeastConfigBaseModel
-from feast.types import (
-    Array,
-    Bytes,
-    FeastType,
-    Float32,
-    Float64,
-    Int32,
-    Int64,
-    Invalid,
-    String,
-)
+from feast.types import Array, Bytes, Float32, Float64, Int32, Int64, Invalid, String
 from feast.usage import log_exceptions_and_usage
 
 logger = logging.getLogger(__name__)
 
+FEAST_MILVUS_CONV_DICT = bidict(
```

Comment:
let's just call it `TYPE_MAPPING` and then have two methods for `get_milvus_type()` and `get_feast_type()`. Since this is be more readable than `bidict.inverse()`